eng/code-quality-reports/src/main/resources/checkstyle/checkstyle-suppressions.xml

```diff
@@ -571,4 +571,8 @@ the main ServiceBusClientBuilder. -->
 
   <suppress checks="com.azure.tools.checkstyle.checks.EnforceFinalFieldsCheck" files="com.azure.search.documents.indexes.models.SynonymMap"/>
 
+  <suppress checks="MissingJavadocMethod" files="com.azure.core.test.http.HttpClientTests"/>
```

Comment:
I'm curious why we need these suppressions? I don't remember adding ones similar to these when writing my own tests... hmm.  I think this is an easy problem to solve by writing some docs.

Reply:
This is because this is a source code file in `azure-core-test` that the `HttpClient` SDKs implement in their test code. Should think about moving this into test code and share it with those libraries using a test-jar.